### PR TITLE
Enable travis, remove random files, fix executable

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@jacob314 Let me push this, if it looks good

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-language: dart # NodeJS is available everywhere
 sudo: false
+language: node_js
 node_js:
-  - "stable"
+  - "6"
```
Comment (jacob314) [inline .travis.yml]: revert this

File: package.json [hunk 2]
```diff
@@ -1,12 +1,7 @@
 {
   "name": "dart_js_facade_gen",
-  "version": "0.0.1",
+  "version": "0.0.2-dev",
```
Comment (jacob314) [inline package.json]: 0.0.2

Comment (kevmoo) [inline package.json]: done

File: .travis.yml (hunk 1 above)
Comment (kevmoo) [inline .travis.yml]: done

**Pull request merged**